### PR TITLE
Update new project section

### PR DESCRIPTION
**Pull request by cunla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,81 +96,79 @@ Django Commons packages.
...
+
+    - [ ] Create a pull-request to `main` branch. This will trigger terraform to plan the changes in the organization to
+      be executed.
+      Review the changes and make sure they align with the request.
+    - [ ] Merge the pull request. This will trigger terraform to apply the changes in the organization.
+    - The expected changes:
+        - [ ] New teams `repo-name`, `repo-name-admins`, `repo-name-committers` with the relevant members based on the
+          repository's description.
+        - [ ] The repository changes are accepted by the project maintainers.
+        - [ ] Repository has two environments: `pypi` and `testpypi`, see example [here][playground-enviroments]
```
Comment (tim-schilling) [inline README.md:157]:
```suggestion
        - [ ] Repository has two environments: `pypi` and `testpypi`, see example [here][playground-environments]
```

File: README.md [hunk 2]
```diff
@@ -96,81 +96,79 @@ Django Commons packages.
...
+
+- [ ] Repo changes:
+    - [ ] (project owner) Create/Update the release GitHub workflow in the repository, example can be
+      found [here][release-gh-workflow]
+    - [ ] Under Actions > General > "Fork pull request workflows from outside collaborators", set "Require approval for
+      first-time contributors"
+
+- [ ] pypi and test-pypi changes:
```
Comment (tim-schilling) [inline README.md:165]:
```suggestion
- [ ] PyPI and Test PyPI changes:
```

File: README.md [hunk 3]
```diff
@@ -96,81 +96,79 @@ Django Commons packages.
...
+    - [ ] Add the release workflow to pypi's package publishing (and test-pypi's package publishing).
```
Comment (tim-schilling) [inline README.md:166]:
```suggestion
    - [ ] Add the release workflow to pypi.org's package publishing (and test.pypi.org's package publishing).
```

File: README.md [hunk 4]
```diff
@@ -196,7 +194,23 @@ The expected changes:
...
+
+[pypi]: https://pypi.org/
+
+[gh-docs-transfer-repo]: https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository
+
+[release-gh-workflow]: https://github.com/django-commons/django-commons-playground/blob/main/.github/workflows/release.yml
+
+[pypi-publishing]: https://test.pypi.org/manage/project/django-tasks-scheduler/settings/publishing/
+
+[playground-enviroments]: https://github.com/django-commons/django-commons-playground/settings/environments
```
Comment (tim-schilling) [inline README.md:216]:
```suggestion
[playground-environments]: https://github.com/django-commons/django-commons-playground/settings/environments
```

**Review by tim-schilling — CHANGES_REQUESTED** [review]
A few small changes, but this looks good.

@cunla what do you think about asking the maintainers to create the release.yml and the test.pypi.org project before scheduling the call? Theoretically, we could also ask them all the various questions to make generating the repo config data easier too.

**Comment by ryancheley** [comment]
> @cunla what do you think about asking the maintainers to create the release.yml and the test.pypi.org project before scheduling the call? Theoretically, we could also ask them all the various questions to make generating the repo config data easier too.

@tim-schilling  I was thinking about this after our call yesterday as well! I think it might be a good idea to ask the maintainers to do that OR to add in an extra 15 - 20 minutes (maybe more 🤷 ) to allow us time to do it on the call with them.

**Comment by tim-schilling** [comment]
My opinion is that we should try to get the information beforehand to keep those meetings as short as we can.

I do think a video will help, but as we optimize the process, it'll keep needing to be updated.

**Comment by cunla** [comment]
I am all for making it more efficient and this is a nice step forward.

Eventually, I hope we can get to a point where maintainers can transfer packages with our involvement only in the background. 
Theoretically, it is already possible: (1) approve PR to become member, (2) approve invite to PyPI, (3) approve PR to add repo to be managed.

Once the process stabilizes, I would like to do another video.

File: README.md [hunk 5]
```diff
@@ -98,82 +98,79 @@ Django Commons packages.
...
-   Review the changes and make sure they align with the request.
-3. Merge the pull request. This will trigger terraform to apply the changes in the organization.
+### Pre Transfer Steps
 
-The expected changes:
+- [ ] Check if the repository meets [inbound requirements][3].
+- [ ] Confirm who will be the admins and maintainers for the repository
+- [ ] Make sure the there are no teams `{repo-name}`, `{repo-name}-admins` and `{repo-name}-committers` in the Django
+  Commons organization. Teams can be viewed [here][teams]. The teams will be created by the terraform apply process.
+- [ ] (project owner) PyPI project owner must add the Django Commons pypi's team members as owners in [PyPI][pypi],
```
Comment (tim-schilling) [inline README.md:109]:
```suggestion
- [ ] (project owner) PyPI project owner must add the Django Commons PyPI Admins as owners in [PyPI][pypi],
```

File: README.md [hunk 6]
```diff
@@ -98,82 +98,79 @@ Django Commons packages.
...
+  and [test-pypi][test-pypi]
+  (TODO: Determine how this works with transferring out of an org and into the Django Commons org)
```
Comment (tim-schilling) [inline README.md:111]:
```suggestion
```

**Review by tim-schilling — APPROVED** [review]

**Pull request merged**